### ISSUE TITLE
rails new mega -j esbuild --css tailwind creates package.json without scripts

### ISSUE THREAD

**Issue by gobijan** [rails/rails#43309]
### Steps to reproduce

Create a new rails 7 app using alpha2 with these options:

```bash
rails new mega -j esbuild --css tailwind 
```

### Expected behavior
Running bin/dev should work (therefore scripts need to be in place in package.json)

### Actual behavior
bin/dev breaks because it does not find scripts in package json.
This is the generated package.json:

```json
{
  "name": "app",
  "private": "true",
  "dependencies": {
    "@hotwired/stimulus": "^3.0.0",
    "@hotwired/turbo-rails": "^7.0.1",
    "autoprefixer": "^10.3.6",
    "esbuild": "^0.13.2",
    "postcss": "^8.3.8",
    "tailwindcss": "^2.2.16"
  }
}
```

### System configuration
**Rails version**: Rails 7 alpha2

**Ruby version**: 3.0.2

**Comment by SkipKayhil** [comment]
What are your `jsbundling-rails` version and `npm/yarn` versions?

It looks like the build script should be getting added here: 
https://github.com/rails/jsbundling-rails/blob/122f356d5d472f91b7efa388c52d97bada4b673d/lib/install/esbuild/install.rb#L5

But I think this depends on npm 7.x

**Comment by gobijan** [comment]
Hey, the jsbundling-rails is as follows in gemfile:
gem "jsbundling-rails", "~> 0.1.0"
and resolves to: 0.1.7

**Comment by gobijan** [comment]
Using the non LTS Version of node (16) seems to make it work. Thx for the hint.
